x-pack/plugins/ml/public/components/anomalies_table/links_menu.js

```diff
@@ -119,7 +119,11 @@ export const LinksMenu = injectI18n(class LinksMenu extends Component {
           // Replace any tokens in the configured url_value with values from the source record,
           // and then open link in a new tab/window.
           const urlPath = replaceStringTokens(customUrl.url_value, record, true);
-          window.open(urlPath, '_blank');
+          if (isKibanaUrl(customUrl) === true) {
```

Comment:
this block of code is repeated a few times, would it be worth moving it to a function in `custom_url_utils.js` called `openURL` or something similar?